community-operators/etcd/etcdoperator.v0.9.4.clusterserviceversion.yaml

```diff
@@ -0,0 +1,353 @@
+            "etcdEndpoints": ["<etcd-cluster-endpoints>"],
+            "storageType":"S3",
+            "s3": {
+              "path": "<full-s3-path>",
+              "awsSecret": "<aws-secret>"
+            }
+          }
+        }
+      ]
+    description: Creates and maintain highly-available etcd clusters on Kubernetes
+spec:
+  displayName: etcd
+  description: |
+    The etcd Operater creates and maintains highly-available etcd clusters on Kubernetes, allowing engineers to easily deploy and manage etcd clusters for their applications.
+
+    etcd is a distributed key value store that provides a reliable way to store data across a cluster of machines. It’s open-source and available on GitHub. etcd gracefully handles leader elections during network partitions and will tolerate machine failure, including the leader.
+
+
+    ### Reading and writing to etcd
+
+    Communicate with etcd though its command line utility `etcdctl` via port forwarding:
+
+        $ kubectl --namespace default port-forward service/example-client 2379:2379
+        $ etcdctl --endpoints http://127.0.0.1:2379 get /
+
+    Or directly to the API using the automatically generated Kubernetes Service:
+
+        $ etcdctl --endpoints http://example-client.default.svc:2379 get /
+
+    Be sure to secure your etcd cluster (see Common Configurations) before exposing it outside of the namespace or cluster.
+
+
+    ### Supported Features
+
+    * **High availability** - Multiple instances of etcd are networked together and secured. Individual failures or networking issues are transparently handled to keep your cluster up and running.
+
+    * **Automated updates** - Rolling out a new etcd version works like all Kubernetes rolling updates. Simply declare the desired version, and the etcd service starts a safe rolling update to the new version automatically.
+
+    * **Backups included** - Create etcd backups and restore them through the etcd Operator.
+
+    ### Common Configurations
+
+    * **Configure TLS** - Specify [static TLS certs](https://github.com/coreos/etcd-operator/blob/master/doc/user/cluster_tls.md) as Kubernetes secrets.
+
+    * **Set Node Selector and Affinity** - [Spread your etcd Pods](https://github.com/coreos/etcd-operator/blob/master/doc/user/spec_examples.md#three-member-cluster-with-node-selector-and-anti-affinity-across-nodes) across Nodes and availability zones.
+
+    * **Set Resource Limits** - [Set the Kubernetes limit and request](https://github.com/coreos/etcd-operator/blob/master/doc/user/spec_examples.md#three-member-cluster-with-resource-requirement) values for your etcd Pods.
+
+    * **Customize Storage** - [Set a custom StorageClass](https://github.com/coreos/etcd-operator/blob/master/doc/user/spec_examples.md#custom-persistentvolumeclaim-definition) that you would like to use.
+  keywords: ['etcd', 'key value', 'database', 'coreos', 'open source']
+  version: 0.9.4
+  maturity: alpha
+  replaces: etcdoperator.v0.9.4
```

Comment:
This should be the name of the previous CSV, ie `etcdoperator.v0.9.2`, in order for upgrades to work.